packages/repository/src/repositories/relation.repository.ts

```diff
@@ -144,3 +145,69 @@ export class DefaultHasManyEntityCrudRepositorys<T extends Entity, ID>
     throw new Error('Method not implemented.');
   }
 }
+
+export class RelationFactory<
+  Source extends Entity,
+  Target extends Entity,
+  TargetRepository extends DefaultCrudRepository<
+    Target,
+    typeof Entity.prototype.id
+  >
+> {
+  constructor(
+    protected readonly sourceInstance: Source,
+    protected readonly relationName: string,
+    protected readonly foreignKey: string | number,
+    protected readonly targetRepository: TargetRepository,
+  ) {
+    this.relationName = relationName || 'hasMany';
```

Comment:
In LB3, when "Customer hasMany orders", the relation _name_ is "orders" and the relation _type_ is "hasMany".

What do you use relationName for? If it's the type of the relation (as hinted by "hasMany" default value), then could you please rename it to `relationType`?

Reply:
Please ignore; I meant to put it as `orders`, but at this point I'm going to spoon feed the values manually, so going to remove it.